daisy/workflow/test_common_test.go

```diff
@@ -102,6 +105,25 @@ func newTestGCEClient() (*compute.TestClient, error) {
 		}
 	}))
 
+	c.GetProjectFn = func(project string) (*compute.Project, error) {
```

Comment:
Cool, I like this. I thought you just didn't like my shorthand p and z variables :).